### PR TITLE
Chore : Fix typo in multiple files (npm)

### PR DESCRIPTION
**Pull request by thapasusheel**
This PR fix the issue in multiple files

```pseudo
idential > identical
exsiting > existing
verions > version
fieled > field
implmentation > implementation
addtion > addition
permisssions > permissions
```

### CODE REVIEW

File: content/cli/v10/commands/npm-sbom.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ SBOM format to use when generating SBOMs.
 * Type: "library", "application", or "framework"
 
 The type of package described by the generated SBOM. For SPDX, this is the
-value for the `primaryPackagePurpose` fieled. For CycloneDX, this is the
+value for the `primaryPackagePurpose` field. For CycloneDX, this is the
```
Comment (wraithgar) [inline content/cli/v10/commands/npm-sbom.md:300]: This file is sourced from https://github.com/npm/cli

File: content/cli/v10/using-npm/config.md [hunk 2]
```diff
@@ -1386,7 +1386,7 @@ SBOM format to use when generating SBOMs.
 * Type: "library", "application", or "framework"
 
 The type of package described by the generated SBOM. For SPDX, this is the
-value for the `primaryPackagePurpose` fieled. For CycloneDX, this is the
+value for the `primaryPackagePurpose` field. For CycloneDX, this is the
```
Comment (wraithgar) [inline content/cli/v10/using-npm/config.md:1389]: https://github.com/npm/cli

File: content/cli/v6/using-npm/changelog.md [hunk 3]
```diff
@@ -289,7 +289,7 @@ redirect_from:
 ### DEPENDENCIES
 
 * [`909cc3918`](https://github.com/npm/cli/commit/909cc39180a352f206898481add5772206c8b65f) `hosted-git-info@2.8.8` ([@darcyclarke](https://github.com/darcyclarke))
-  * [`5038b1891`](https://github.com/npm/hosted-git-info/commit/5038b1891a61ca3cd7453acbf85d7011fe0086bb) fix: regression in old node versions w/ respect to url.URL implmentation
+  * [`5038b1891`](https://github.com/npm/hosted-git-info/commit/5038b1891a61ca3cd7453acbf85d7011fe0086bb) fix: regression in old node versions w/ respect to url.URL implementation
```
Comment (wraithgar) [inline content/cli/v6/using-npm/changelog.md:292]: This is part of the npm v6 changelog and it is what it is.

File: content/cli/v6/using-npm/scripts.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ To create "pre" or "post" scripts for any scripts defined in the `"scripts"` sec
 
 ### Life Cycle Scripts
 
-There are some special life cycle scripts that happen only in certain situations. These scripts happen in addtion to the "pre" and "post" script.
+There are some special life cycle scripts that happen only in certain situations. These scripts happen in addition to the "pre" and "post" script.
```
Comment (wraithgar) [inline content/cli/v6/using-npm/scripts.md:34]: This field is sourced from https://github.com/npm/cli in the old v6 branch and is likely going to remain as it is.

File: content/cli/v7/using-npm/changelog.md [hunk 5]
```diff
@@ -1581,7 +1581,7 @@ redirect_from:
 
 * [`ba1adef42`](https://github.com/npm/cli/commit/ba1adef4292123e87e26b59e0c6fd6f5ff1fe775)
   [#2760](https://github.com/npm/cli/issues/2760)
-  chore(docs): capitalize all Instaces of "package"
+  chore(docs): capitalize all Instances of "package"
```
Comment (wraithgar) [inline content/cli/v7/using-npm/changelog.md:1584]: This is from the v7 changelog and is going to remain as-is

File: content/cli/v8/using-npm/dependency-selectors.md [hunk 6]
```diff
@@ -13,7 +13,7 @@ redirect_from:
 
 ### Description
 
-The [`npm query`](/cli/v8/commands/npm-query) commmand exposes a new dependency selector syntax (informed by & respecting many aspects of the [CSS Selectors 4 Spec](https://dev.w3.org/csswg/selectors4/#relational)) which:
+The [`npm query`](/cli/v8/commands/npm-query) command exposes a new dependency selector syntax (informed by & respecting many aspects of the [CSS Selectors 4 Spec](https://dev.w3.org/csswg/selectors4/#relational)) which:
```
Comment (wraithgar) [inline content/cli/v8/using-npm/dependency-selectors.md:16]: This is sourced from the v8 branch of https://github.com/npm/cli

File: content/cli/v8/using-npm/scope.md [hunk 7]
```diff
@@ -98,7 +98,7 @@ Publishing to a scope, you have two options:
 If publishing a public module to an organization scope, you must
 first either create an organization with the name of the scope
 that you'd like to publish to or be added to an existing organization
-with the appropriate permisssions. For example, if you'd like to 
+with the appropriate permissions. For example, if you'd like to
```
Comment (wraithgar) [inline content/cli/v8/using-npm/scope.md:101]: This is sourced from the v8 branch of https://github.com/npm/cli

**Comment by thapasusheel** [comment]
@wraithgar I didnot get what you are trying to say.

Can you please advise me what to next so that my PR will be valid?

**Comment by lukekarrys** [comment]
@thapasusheel To land this, the changes in `content/cli/` need to be reverted. Those fixes should be made in https://github.com/npm/cli.

**Comment by lukekarrys** [comment]
I'm going to close this, but feel free to reopen or to open a new PR to https://github.com/npm/cli.

**Pull request closed without merging**